### ISSUE TITLE
How to get weights from a merge layer in keras?

### ISSUE THREAD

**Issue by natmourajr** [fchollet/keras#4628]
I'm  trying to train each part of a model separately, like in this piece of code:

from keras.layers import Merge

left_branch = Sequential()
left_branch.add(Dense(2, input_dim=4,name='left'))

right_branch = Sequential()
right_branch.add(Dense(1, input_dim=4))

merged = Merge([left_branch, right_branch], mode='concat')

final_model = Sequential()
final_model.add(merged)
final_model.add(Dense(1, activation='softmax'))

How could I get weights from the right branch after training process?
Thanks

**Comment by mabirck** [comment]
I don't know if there is a beauty and god practice way, but a manul way would be open manually your model.

**I opened step by step just to be in didactic way** 
![screenshot from 2016-12-07 15-01-41](https://cloud.githubusercontent.com/assets/13804708/20977997/62e14eda-bc8e-11e6-8e57-1e8fa853d024.png)

**Comment by natmourajr** [comment]
Thank you very much @mabirck, it works exactly as I expect.
